README.md

```diff
@@ -1,50 +1,25 @@
 
-## How to track what we're doing, and how you can be involved!
+- Guide: https://frontend.18f.gov/
 
-We use issues in this repo to track work. They are accessible in a nice visual, prioritized form on [our waffle board](https://waffle.io/18F/frontend).
+## How to track what we're doing, and how you can be involved!
 
-Also, the Frontend Guild Roadmap is in [the wiki](https://github.com/18F/frontend/wiki), and research into what the guild should be doing to support frontend practice at 18F is kept in [this branch](https://github.com/18F/frontend/tree/research).
+We use issues in this repo to track work.
 
 ### How Guild best practices are developed
 
 1. Team members suggest needs in issues, and guild leads identify issues through [research](https://github.com/18F/frontend/tree/research).
-2. If resolving these issues requires documentation, a draft document is created in an internal Google Doc that is open to comment and announced to #frontend for contribution. NB: You can see all in-process docs in [this internal folder](https://drive.google.com/drive/u/1/#folders/0B84F26FpUP0lR1B2VVNGSi1MMVk/0B0C6PKlzps2JV3pqX3NJdm5WejA/0B5HeQa_YQ6-VTTlkVEFNZ2VWZEU/0B2CjDILjK8_jfmp1c2ZJM2d0eEtGSHFEeS1CenlHWEQ0S01jcWJfZXNObElUQV9Yei0wZ2s).
-3. Once a relatively clear and stable version emerges, an alert is posted to #dev, #frontend and #design for more review and comment.
+2. If resolving these issues requires documentation, a draft document is created in an internal Google Doc that is open to comment and announced to #g-frontend for contribution. NB: You can see all in-process docs in [this internal folder](https://drive.google.com/drive/u/1/#folders/0B84F26FpUP0lR1B2VVNGSi1MMVk/0B0C6PKlzps2JV3pqX3NJdm5WejA/0B5HeQa_YQ6-VTTlkVEFNZ2VWZEU/0B2CjDILjK8_jfmp1c2ZJM2d0eEtGSHFEeS1CenlHWEQ0S01jcWJfZXNObElUQV9Yei0wZ2s).
+3. Once a relatively clear and stable version emerges, an alert is posted to #dev, #g-frontend and #design for more review and comment.
 4. Once comments are resolved, the guide is posted publicly (location dependent on content).
-5. The best practice or guide is announced to the entire 18F Team via #news and email.
-
-### How to create a new guide
-
-1. Fork or clone the frontend repo. `git clone git@github.com:18F/frontend.git`
-2. Create a new branch for your guide, prefixed with `draft-`. `git checkout -b draft-a_new_guide`
-3. Create a markdown(md) file in the pages directory for your guide. `touch pages/draft-a_new_guide.md`
-4. After editing add, commit and push changes. `git add pages && git commit && git push origin draft-a_new_guide`
-5. Create a pull request on github.
-6. Label your pull request with `draft`.
-7. Post in #frontend slack channel for comment.
-
-If ready for publishing, you or someone else can move your guide to the 18f-pages branch so it appears on frontend.18f.gov.
+5. The best practice or guide is announced to TTS via #news and email.
```

Comment:
What types of announcements are we making to #news when we have updates to the guide?

Reply:
Tis a good question. I wasn't aware that this was how we were supposed to do this until just now, and I'd like to ask the group if they think it would be a useful thing to do. Possibly it would be helpful given the expanded audience.